### DISCUSSION TITLE
My build! (First electronic engineering project)

### DISCUSSION THREAD

**Discussion by Beatrix-droid** [martinwoodward/octolamp#16] [Show and tell]
Here is my build! I am very pleased with it! As a total beginner in electronic engineering projects, I found this video of someone configuring led light strips using the same software and hardware listed in this project to be extremely helpful: leaving it here so that other beginners may find it helpful:

https://youtu.be/iEtS3rLzdZ0

Here are some pictures of my build:![image](https://user-images.githubusercontent.com/79793130/221183957-32bae468-da9c-4f7f-b1d0-b05d1aa124c4.jpeg)
https://user-images.githubusercontent.com/79793130/221184097-d55a5d40-4f89-4272-8e1f-b66a8162da48.MOV

**Comment by martinwoodward** [comment]
Great job @Beatrix-droid - and thanks for the video link. Great tip
